### DISCUSSION TITLE
BUG(UI - Notifications): Select All with minus (-) icon still selects all notifications

### DISCUSSION THREAD

**Discussion by kindoflew** [community/community#22388] [New to GitHub]
<p>When I manually select several notifications, the <code>Select All</code> checkbox contains a minus sign icon. If I click that to un-select the notifications I just marked, it will select all notification checkboxes on the screen. I’m assuming it’s supposed to un-select all checkboxes.</p>
<p>Steps to reproduce:</p>
<ul>
<li>Go to <a href="http://github.com/notifications" rel="noopener nofollow ugc">github.com/notifications</a>
</li>
<li>Select one or more notifications in the list</li>
<li>Click the Minus (<code>-</code>) icon in the Select All checkbox</li>
<li>All notifications are selected</li>
</ul>

**Comment by Katsute** [comment, marked as answer]
<p>That is not a bug the <code>[-]</code> checkbox means a mixed response <a href="https://www.w3.org/TR/2017/WD-wai-aria-practices-1.1-20170628/examples/checkbox/checkbox-2/checkbox-2.html" class="inline-onebox" rel="noopener nofollow ugc">Checkbox Example (Mixed State) | WAI-ARIA Authoring Practices 1.1</a></p>

**Comment by kindoflew** [comment]
<p>Oh, cool – TIL, thanks! The only time I’ve seen that icon/pattern was in my email where it behaves as I assumed.</p>
<p>So I agree, not a bug. But also GitHub’s implementation of it does not follow that pattern because: 1) you can’t cycle through all three states and 2) it doesn’t represent the various states with <code>aria-checked</code> attributes. So, I’ll close this and possibly open a feature request thread for that then.</p>
<p>Thanks again!</p>
